app/src/main/res/values-es/strings.xml

```diff
@@ -1,26 +1,32 @@
-<?xml version='1.0' encoding='utf-8'?>
+<?xml version="1.0" encoding="utf-8"?>
+<!--Generated by crowdin.com-->
 <resources>
-  <string name="app_name">Open Food</string>
   <string name="hello_world">¡Hola mundo!</string>
   <string name="action_settings">Configuración</string>
   <string name="action_about">Acerca de</string>
   <string name="action_discover">Descubrir</string>
   <string name="contribute">Cómo contribuir</string>
   <string name="action_find">Buscar</string>
   <string name="text_barcode">Escribe el código de barras :</string>
+  <!-- Nav Drawer Product Menu Items -->
   <string-array name="nav_drawer_items_product">
     <item>Resumen</item>
     <item>Ingredientes</item>
     <item>Nutrición</item>
     <item>Tabla Nutricional</item>
   </string-array>
+  <!-- Text units -->
   <string-array name="units_array">
-    <item>g</item>
+    <item> </item>
```

Comment:
It looks like it also empty in **values/strings.xml**
Looks like it is to ensure the dropdown is empty by default